thirdparty/runtime_environments.go

```diff
@@ -347,6 +367,88 @@ func stringToTime(timeInitial string) (time.Time, error) {
 	return time.Unix(timestamp, 0), nil
 }
 
+// Image stores information about an image including its name, version_id, kernel, ami, and its last deployed time.
+type Image struct {
+	Name         string
+	VersionID    string
+	Kernel       string
+	LastDeployed time.Time
+	AMI          string
+}
+
+// getNameFromOSInfo uses the provided AMI and name (exact match) arguments to filter the image information.
+func (c *RuntimeEnvironmentsClient) getNameFromOSInfo(ctx context.Context, ami string, name string) (string, error) {
+	optsOS := OSInfoFilterOptions{
+		AMI:  ami,
+		Name: fmt.Sprintf("^%s$", name),
+	}
+	resultOS, err := c.GetOSInfo(ctx, optsOS)
+	if err != nil {
+		return "", errors.Wrap(err, "getting OS info")
```

Comment:
you can probably just call these values `opts` and `result` because it's clear what we're operating on based on the function name